index.js

```diff
@@ -38,20 +38,31 @@ searchBtnCmp.addEventListener("click", function() {
             tempUnitCmp.innerHTML = "&#8451";
             newWindVal.innerHTML = Math.round(data.wind.speed);
             weatherWindCmp.appendChild(newWindVal);
-            let weatherIcon = data.weather[0].icon;
-            let icons = new Skycons({ "color": "white" });
-            icons.set("weatherIcon", Skycons.CLEAR_DAY);
-            icons.set("weatherIcon", Skycons.CLEAR_NIGHT);
-            icons.set("weatherIcon", Skycons.PARTLY_CLOUDY_DAY);
-            icons.set("weatherIcon", Skycons.PARTLY_CLOUDY_NIGHT);
-            icons.set("weatherIcon", Skycons.CLOUDY);
-            icons.set("weatherIcon", Skycons.RAIN);
-            icons.set("weatherIcon", Skycons.SLEET);
-            icons.set("weatherIcon", Skycons.SNOW);
-            icons.set("weatherIcon", Skycons.WIND);
-            icons.set("weatherIcon", Skycons.FOG);
+            /* Trying to retrieve animated icon...Not Working */
+            let openWeatherIcon = data.weather[0].icon;
+            console.log(openWeatherIcon);
+            let skycons = new Skycons({ 'color': 'white' });
 
-            switch (weatherIcon) {
+            var iconList = [
+                "CLEAR_DAY",
+                "CLEAR_NIGHT",
+                "PARTLY_CLOUDY_DAY",
+                "PARTLY_CLOUDY_NIGHT",
+                "CLOUDY",
+                "RAIN",
+                "SLEET",
+                "SNOW",
+                "WIND",
+                "FOG"
+            ];
+            console.log(icons);
+            for (let i = 0; i < iconList.length; i++) {
```

Comment:
looks like you don't need to set those icons upfront. So you can get rid of `iconsList` and this for-loop. Switch-case should do all needed work.

Reply:
i deleted iconsList and for loop,but still im getting wrong city name and this is becoming tricky

Reply:
icon is also not showing up

Reply:
@Vidhya-eng  please push your changes so I can check them.

Reply:
now the icons are showing up,i discarded skicons used svg animated icons